lib/services/updaters/ratings.rb

```diff
@@ -0,0 +1,73 @@
+module Services
+  module Updaters
+    class Ratings < Services::Base
+      attribute :current_app
+      attribute :shop_type
+      attribute :country
+      attribute :start_date
+      attribute :device
+      attribute :id
+
+      private
+
+      def perform
+        load_by_android
+        load_by_ios
+      end
+
+      def load_by_android
+        @shop_type = 'android'
+        @device = nil
+        @id = current_app.android_app_id
+        update_app
+      end
+
+      def load_by_ios
+        @shop_type = 'ios'
+        @device = 'iphone'
+        @id = current_app.apple_app_id
+        update_app
+      end
+
+      def update_app
+        TargetCountry.pluck(:country_name).each do |country|
+          @country = country
+          date
+          next if @start_date > Time.zone.today - 1
```

Comment:
используй аксесоры, использование инстанс переменных может привести к багам, так как при опечатке возвращают просто  nil
```suggestion
          next if start_date > Time.zone.today - 1
```

Reply:
Косяк, не заметил